docs/diagrams/GetInpatientSequenceDiagram.puml

```diff
@@ -0,0 +1,69 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":GetCommandParser" as GetCommandParser LOGIC_COLOR
+participant "g:GetInpatientCommand" as GetInpatientCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("get /inp")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("get /inp")
+activate AddressBookParser
+
+create GetCommandParser
+AddressBookParser -> GetCommandParser
+activate GetCommandParser
+
+GetCommandParser --> AddressBookParser
+deactivate GetCommandParser
+
+AddressBookParser -> GetCommandParser : parse("/inp")
+activate GetCommandParser
+
+create GetInpatientCommand
+GetCommandParser -> GetInpatientCommand
+activate GetInpatientCommand
+
+GetInpatientCommand --> GetCommandParser : g
```

Comment:
Might be missing contextual clues here, what does the `g` signify?

Reply:
I do not believe there is any variable name attached to the instance of `GetInpatientCommand` in our actual code.

`g` is only written here to illustrate that the instance of `GetInpatientCommand` is returned from it's creation to the `GetCommandParser` and back to the `AddressBookParser` and finally back to the `LogicManager`.

I believe this would be less ambiguous as return arrows without the variable name `g` might raise some questions on what is returned from each method call.